src/clarity-angular/popover/dropdown/dropdown-toggle.ts

```diff
@@ -30,7 +30,7 @@ export class DropdownToggle {
     }
 
     get active(): boolean {
-        return this._dropdown.open;
+        return this._dropdown.ifOpenService.open;
     }
 
     @HostListener("click")
```

Comment:
Actually this getter is only being used by its own template to set the active class on the host element.. I've made changes so that it gets the value directly from the injected service.